### PR TITLE
Product details slider

### PR DESCRIPTION
**Pull request by vlevik**
loom still doesn't work, so I added a few screenshots here
Знімок екрана 2023-12-19 101225
Знімок екрана 2023-12-19 101244
Знімок екрана 2023-12-19 101257

### CODE REVIEW

File: src/modules/ProductDetailsPage/ProductDetailsPage.tsx [hunk 1]
```diff
@@ -1,26 +1,35 @@
...
 
-// import styles from './ProductDetailsPage.module.scss';
+import styles from './ProductDetailsPage.module.scss';
 
 import { Breadcrumbs } from '../shared/Breadcrumbs';
 import { BackButton } from '../shared/BackButton';
 import { ProductDetailsSlider } from './components/ProductDetailsSlider';
+import { PhoneDetail } from '../../types/PhoneDetail';
+import { getPhoneDetail } from '../../api/service';
 // import { ProductSlider } from '../shared/ProductSlider';
```
Comment (dsfreedom) [inline src/modules/ProductDetailsPage/ProductDetailsPage.tsx:11]: Dont forget to remove it if you don't need it)

File: src/modules/ProductDetailsPage/components/ProductDetailsSlider/Arrows.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styles from './ProductDetailsSlider.module.scss';
+
+export const NextArrow: React.FC = () => {
+  return (
+    <>
```
Comment (dsfreedom) [inline src/modules/ProductDetailsPage/components/ProductDetailsSlider/Arrows.tsx:6]: No need in a fragment component if you have only one child inside

File: src/modules/ProductDetailsPage/components/ProductDetailsSlider/Arrows.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      <div
+        className={styles['slick-next-phone']}
+      />
+    </>
+  );
+};
+
+export const PrevArrow: React.FC = () => {
+  return (
+    <>
```
Comment (dsfreedom) [inline src/modules/ProductDetailsPage/components/ProductDetailsSlider/Arrows.tsx:16]: Same story, fragment component is redundant

File: src/modules/shared/ProductSlider/slick-theme.scss [hunk 4]
```diff
@@ -0,0 +1,132 @@
+@charset "UTF-8";
+@import '../../../styles/blocks/grid';
+@import '../../../styles/utils/variables/colors';
+
+/* Arrows */
```
Comment (dsfreedom) [inline src/modules/shared/ProductSlider/slick-theme.scss:5]: Don't forget about comments)

**Review by dsfreedom — APPROVED** [review]

**Review by DenysShurubura — APPROVED** [review]

**Pull request merged**